src/core/components/accordion/accordionItem/accordionItem.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { forwardRef, type ReactNode } from 'react';
+import { AccordionContainer, type IAccordionContainerProps } from '../accordionContainer/accordionContainer';
+import { AccordionItem } from './accordionItem';
+
+describe('<AccordionItem /> component', () => {
+    const MockChildren = forwardRef<HTMLButtonElement, { children?: ReactNode }>((props, ref) => (
+        <button ref={ref}>{props.children}</button>
+    ));
+    MockChildren.displayName = 'MockHeader';
+
+    const createTestComponent = (values?: Partial<IAccordionContainerProps>) => {
+        const defaultProps: IAccordionContainerProps = {
+            ...values,
+        };
+        return (
+            <AccordionContainer {...defaultProps}>
+                <AccordionItem value="item-1">
+                    <MockChildren>Mock Children 1</MockChildren>
+                </AccordionItem>
+                <AccordionItem value="item-2">
+                    <MockChildren>Mock Children 2</MockChildren>
+                </AccordionItem>
+            </AccordionContainer>
```

Comment:
The `createTestComponent` function should only render the minimum testing setup for the unit tests. The unit tests should also not rely on the properties / children set here as each unit test should be independent from each other.